tools/integration_tests/emulator_tests/proxy_server/operation_manager.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package proxy_server
+
+import (
+	"log"
+	"sync"
+)
+
+type OperationManager struct {
+	retryConfigs map[RequestType][]RetryConfig
+	mu           sync.Mutex
+}
+
+func NewOperationManager(config Config) *OperationManager {
+	rc := make(map[RequestType][]RetryConfig)
+	om := &OperationManager{
+		retryConfigs: rc,
+	}
+	for _, retryConfig := range config.RetryConfig {
+		om.addRetryConfig(retryConfig)
+	}
+	log.Printf("%+v\n", om)
```

Comment:
Is there any way to add these print statement only in debug mode?

Reply:
I'll add main.go file in upcoming PR, their I'll introduce this flag and cover this comment.